account_statement_base/tests/test_account_statement_base.py

```diff
@@ -1,16 +1,15 @@
 from odoo import Command
 from odoo.tests import tagged
 
-from odoo.addons.account_reconcile_model_oca.tests.common import (
-    TestAccountReconciliationCommon,
-)
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
 
 
 @tagged("post_install", "-at_install")
-class TestReconciliationWidget(TestAccountReconciliationCommon):
+class TestReconciliationWidget(AccountTestInvoicingCommon):
```

Comment:
TestReconciliationWidget does not reflect the purpose of the test, can you improve it?